packages/opentelemetry-resource-util/test/index.test.ts

```diff
@@ -124,6 +124,262 @@ describe('mapOtelResourceToMonitoredResource', () => {
       },
     },
 
+    {
```

Comment:
Thanks for updating the tests

Since these cases (along with the GKE one) are very repetitive and prone to typos, I wouldn't be opposed to templating it. Wdyt?

```ts
...[{on: "azure_aks", platform: "azure_aks"}, {on: "non-cloud", platform: undefined}, ...].map(({on, platform}) => ({
  "title": `should map to k8s_container on ${on}`,
  // ...
})),
```

Reply:
I thought too. So I fix it. https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/724/commits/008c3c868698bd2313e2638f6624cefcafdeb0bb